scheduler/plugin/plugin.go

```diff
@@ -20,17 +20,14 @@ import (
 	"context"
 	"fmt"
 	"os"
-	"sync"
 	"sync/atomic"
 	"time"
 
+	"github.com/tetratelabs/wazero"
 	v1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/runtime"
-	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/kubernetes/pkg/scheduler/framework"
 	frameworkruntime "k8s.io/kubernetes/pkg/scheduler/framework/runtime"
-
-	"github.com/tetratelabs/wazero"
 )
 
 const PluginName = "wasm"
```

Comment:
all logic in this file is the same, I only implemented the plugin interfaces, stubbed with TODO.